### PR TITLE
Modified project of individual contribution

### PR DESCRIPTION
**Pull request by Sans1509**
I have made the changes , please review it.

### CODE REVIEW

File: Sanskriti/linear_regression/.idea/.gitignore
Comment (garg-tanishka) [inline Sanskriti/linear_regression/.idea/.gitignore:1]: remove .idea folder

File: Sanskriti/linear_regression/src/linear_regression/pipeline/constant.py
Comment (garg-tanishka) [inline Sanskriti/linear_regression/src/linear_regression/pipeline/constant.py:1]: Move constants file into utils directory.

File: Sanskriti/linear_regression/src/linear_regression/model_validation/validation.py
Comment (garg-tanishka) [inline Sanskriti/linear_regression/src/linear_regression/model_validation/validation.py:1]: Remove these empty files/directories

File: Sanskriti/linear_regression/src/linear_regression/model_training/training.py
Comment (garg-tanishka) [inline Sanskriti/linear_regression/src/linear_regression/model_training/training.py:1]: why do we have two python files as model_training and training?

File: Sanskriti/linear_regression/src/linear_regression/pipeline/constant.py
Comment (Sans1509) [reply Sanskriti/linear_regression/src/linear_regression/pipeline/constant.py:1]: done

File: Sanskriti/linear_regression/src/linear_regression/model_validation/validation.py
Comment (Sans1509) [reply Sanskriti/linear_regression/src/linear_regression/model_validation/validation.py:1]: done

File: Sanskriti/linear_regression/src/linear_regression/preprocessing/processing.py
[the remark this answers is not in the document]
Comment (Sans1509) [reply Sanskriti/linear_regression/src/linear_regression/preprocessing/processing.py:1]: done

File: Sanskriti/linear_regression/src/linear_regression/model_training/training.py
Comment (Sans1509) [reply Sanskriti/linear_regression/src/linear_regression/model_training/training.py:1]: done

File: Sanskriti/linear_regression/.idea/.gitignore
Comment (Sans1509) [reply Sanskriti/linear_regression/.idea/.gitignore:1]: done

**Pull request merged**